content/en/docs/concepts/wasm/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WebAssembly (Wasm) Plugins
+description: Describes Istio's WebAssembly Plugin system.
+weight: 25
+keywords: [wasm,webassembly,emscripten,extension,plugin,filter]
+---
+
+WebAssembly is a sandboxing technology which can be used to extend the Istio proxy (Envoy).  Going forward Istio extensions will be written as proxy-Wasm plugins using the WebAssembly (Wasm) sandbox API. Mixer will be eliminated from the Istio architecture and all Mixer adapters must be eventually rewritten as proxy plugins.  
```

Comment:
last sentence ends in proxy plugin. Should it be proxy-Wasm plugin?

Reply:
Can we add a small paragraph before this? "What is WebAssembly?" with a link to webassembly.org or some other authoritative source.